samtranslator/plugins/application/serverless_app_plugin.py

```diff
@@ -104,6 +105,10 @@ def on_before_transform_template(self, template_dict):
 
             if key not in self._applications:
                 try:
+                    if not RegionConfiguration().is_sar_supported():
```

Comment:
Do we need to create a class instance to use class method?

Reply:
Indeed, `is_apigw_edge_configuration_supported` is used like this `RegionConfiguration.is_apigw_edge_configuration_supported()`

Reply:
Updated.